src/node_crypto.cc

```diff
@@ -1236,11 +1236,14 @@ int SecureContext::TicketKeyCallback(SSL* ssl,
                  kTicketPartSize).ToLocalChecked(),
     Boolean::New(env->isolate(), enc != 0)
   };
-  Local<Value> ret = node::MakeCallback(env,
+
+  // TODO(addaleax): `SecureContext` should maybe inherit from AsyncWrap?
+  Local<Value> ret = node::MakeCallback(env->isolate(),
                                         sc->object(),
                                         env->ticketkeycallback_string(),
                                         arraysize(argv),
-                                        argv);
+                                        argv,
+                                        0).ToLocalChecked();
```

Comment:
See https://github.com/nodejs/node/issues/13176, I don’t think we need to actually care about this